src/components/RecipeDetails.tsx

```
@@ -122,11 +122,11 @@ const RecipeDetails: React.FC = () => {
                     {hasSubRecipes && adjustedSubRecipes.map((subRecipe, index) => (
                         <div key={index} className="mb-6">
                             <h3 className="text-xl font-semibold mb-2">{subRecipe.title}</h3>
-                            <p>{subRecipe.instructions}</p>
+                            <div dangerouslySetInnerHTML={{ __html: subRecipe.instructions }} />
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)